AsyncLibrary/Fibonacci.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AsyncLibrary
+{
+    public static class Fibonacci
+    {
+        public static int Fib(int i)
+        {
+            Thread.Sleep(10);
+
+            if (i == 1)
+                return 0;
+
+            if (i == 2)
+                return 1;
+
+            return Fib(i - 2) + Fib(i - 1);
+        }
+
+        public static async Task FibAsync()
+        {
+            var fib_5  = Task.Run(() => { return Fib(5); });
+            var fib_7 = Task.Run(() => { return Fib(7); });
+            var fib_11 = Task.Run(() => { return Fib(11); });
```

Comment:
1. await keyword can be used right here before Task.Run(...);
2. () => { return Fib(); } can be simplified to just () => Fib(7)